test/api.js

```diff
@@ -0,0 +1,69 @@
+var bodyParser = require('koa-bodyparser');
+var route = require('koa-route');
+var path = require('path');
+var ejs = require('ejs');
+
+module.exports = api;
+
+/**
+ * Recurly.js API mock
+ * @param {Object} app Koa app
+ */
+function api (app) {
+  app.use(cors);
+  app.use(contentTypeFix);
+  app.use(bodyParser());
+  app.use(route.get('/plans/:plan_code', json));
+  app.use(route.get('/plans/:plan_code/coupons/:coupon_code', json));
+  app.use(route.post('/token', json));
+  app.use(route.get('/token', json));
+  app.use(route.get('/tax', json));
+  app.use(route.get('/paypal/start', postMessage));
+  app.use(route.get('/relay', relay));
+}
+
+/**
+ * Utility functions.
+ */
+
+function* json () {
+  var data = fixture.apply(this);
+  if (this.query.callback) {
+    this.body = [this.query.callback, '(', JSON.stringify(data), ')'].join('');
```

Comment:
It might be worthwhile to set the Content-Type to application/javascript, but this is MVP for jsonp. If we get lots of console warnings, it might be good to just set the Content-Type and be done with it.